src/uct/ib/mlx5/ece.h

```diff
@@ -0,0 +1,79 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2021.  ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#ifndef UCT_ECE_H_
+#define UCT_ECE_H_
+
+
+#include <uct/ib/base/ib_log.h>
+#include <uct/ib/base/ib_device.h>
+#include <ucs/debug/log.h>
+#include <ucs/type/status.h>
+
+#define UCT_IB_MLX5_VENDOR_ID             0x15b3
```

Comment:
check if we use in other places as constant, but not include mlx5 from base code